packages/python-ccrawl/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-ccrawl
```

Comment:
pkgbase used only when you do 2 in 1 package like py2 + py3 lib, so if only py3 no pkgbase